.travis.yml

```diff
@@ -1,10 +1,11 @@
 language: node_js
 node_js:
-    - "0.10"
-    - "0.12"
-    - "iojs-v2.5.0"
+  - "0.10"
+  - "0.12"
+  - "4.2.4"
+  - "5.1"
```

Comment:
I put that one because we test against the same one in RESTBase too. But, overall, I think it's more important to test with *a* 5.x version, *which one* is not as important.